is-array/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for is-array 1.0
+// Project: https://github.com/retrofox/is-array
+// Definitions by: Pine Mizune <https://github.com/pine>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = isArray;
+declare function isArray(val?: any): boolean;
```

Comment:
The type of the polyfill should agree with that in `lib.d.ts`:
https://github.com/Microsoft/TypeScript/blob/master/lib/lib.d.ts#L1310